error in setup command: 'tests_require' must be a string or list of strings containing valid project/version requirement specifiers; Expected end or semicolon (after version specifier)
    pytz>=2010h
        ~~~~~~^
diff --git a/test-requirements.txt b/test-requirements.txt
index bc8d8d2..f997bf9 100644
--- a/test-requirements.txt
+++ b/test-requirements.txt
@@ -27,4 +27,4 @@ scripttest
 # NOTE(rpodolyaka): This version identifier is currently necessary as
 #                   pytz otherwise does not install on pip 1.4 or higher
 pylint
-pytz>=2010h
+pytz>=2010
